quickwit-config/src/config.rs

```diff
@@ -306,16 +310,39 @@ impl QuickwitConfig {
     /// Returns the gRPC listen address of the node, i.e. the socket address on which the gRPC
     /// service listens for TCP connections.
     pub async fn grpc_listen_addr(&self) -> anyhow::Result<SocketAddr> {
-        get_socket_addr(&(self.listen_address.as_str(), self.grpc_listen_port())).await
+        self.listen_address
+            .with_port(self.grpc_listen_port())
+            .to_socket_addr()
+            .await
+    }
+
+    /// Returns the advertise
+    pub fn advertise_addr(&self) -> anyhow::Result<Host> {
+        if let Ok(advertise_address) = env::var("QW_ADVERTISE_ADDRESS") {
```

Comment:
Is it common to have override in that order
ENV > CLI?

I think I usually expect CLI > ENV

Reply:
This case is ENV > Config file.